ui/apps/platform/src/Containers/Collections/CollectionForm.tsx

```diff
@@ -268,44 +294,80 @@ function CollectionForm({
                             </FlexItem>
                         </Flex>
                         <Divider component="div" />
-                        <Flex
-                            className="pf-u-background-color-200 pf-u-p-lg"
-                            spaceItems={{ default: 'spaceItemsMd' }}
-                            direction={{ default: 'column' }}
-                        >
-                            <Card>
-                                <CardBody>
+                        <Form>
+                            <Flex
+                                className="pf-u-background-color-200 pf-u-p-lg"
+                                spaceItems={{ default: 'spaceItemsMd' }}
+                                direction={{ default: 'column' }}
+                            >
+                                <div className="pf-u-background-color-100 pf-u-p-lg">
                                     <Title headingLevel="h2">Collection details</Title>
-                                </CardBody>
-                            </Card>
-                            <Card>
-                                <CardBody>
-                                    <Title headingLevel="h2">Add new collection rules</Title>
-                                    <RuleSelector />
-                                    <RuleSelector />
-                                    <RuleSelector />
-                                </CardBody>
-                            </Card>
-                            <Card>
-                                <CardBody>
+                                </div>
+
+                                <Flex
+                                    className="pf-u-background-color-100 pf-u-p-lg"
+                                    direction={{ default: 'column' }}
+                                    spaceItems={{ default: 'spaceItemsMd' }}
+                                >
+                                    <Title className="pf-u-mb-xs" headingLevel="h2">
+                                        Add new collection rules
+                                    </Title>
+                                    <p>
+                                        Target deployments via selector rules. Regular expressions
```

Comment:
Off the subject of this pull request, here is a plain alternative to consider:

Select collections of deployments via rules. You can use regular expressions (RE2).

* Target is more often a noun than a verb. For example, online dictionary, thesaurus, and translate.
* The distinction between selector rules and fields is not clear.

Reply:
Alan has suggested "Select deployments via rules. You can use regular expressions (RE2 syntax).", which removes the usage of "collections" in the definition of what a collection is, and is more explicit about RE2 _syntax_.

He also mentioned linking the text to the RE2 docs page, or having more info on tooltip. My first thought was to do the former, but I couldn't find a single other instance of an external link in the app. Is this an intentional choice, possibly due to offline environments?